shopinvader_customer_activity_date/__manifest__.py

```diff
@@ -0,0 +1,19 @@
+# Copyright 2022 Camptocamp SA (https://www.camptocamp.com).
```

Comment:
I'd remove the `_date` from this module name to ease further extension

Reply:
agree, module renamed!